src/main/java/org/broadinstitute/hellbender/tools/walkers/sv/SVAnnotateEngine.java

```diff
@@ -0,0 +1,734 @@
+package org.broadinstitute.hellbender.tools.walkers.sv;
+
+import com.google.common.annotations.VisibleForTesting;
+import htsjdk.samtools.SAMSequenceDictionary;
+import htsjdk.variant.variantcontext.Allele;
+import htsjdk.variant.variantcontext.VariantContext;
+import htsjdk.variant.variantcontext.VariantContextBuilder;
+import org.apache.commons.compress.utils.Sets;
+import org.broadinstitute.hellbender.exceptions.UserException;
+import org.broadinstitute.hellbender.tools.spark.sv.utils.GATKSVVCFConstants;
+import org.broadinstitute.hellbender.tools.spark.sv.utils.SVUtils;
+import org.broadinstitute.hellbender.utils.SVInterval;
+import org.broadinstitute.hellbender.utils.SVIntervalTree;
+import org.broadinstitute.hellbender.utils.SimpleInterval;
+import org.broadinstitute.hellbender.utils.codecs.gtf.GencodeGtfFeature;
+import org.broadinstitute.hellbender.utils.codecs.gtf.GencodeGtfTranscriptFeature;
+
+import java.util.*;
+
+public class SVAnnotateEngine {
+    private final int maxBreakendLen;
+    private final SVIntervalTree<GencodeGtfTranscriptFeature> gtfIntervalTree;
+    private final SVIntervalTree<String> promoterIntervalTree;
+    private final SVIntervalTree<String> nonCodingIntervalTree;
+    private final SVIntervalTree<String> transcriptionStartSiteTree;
```

Comment:
```suggestion
    private final GTFIntervalTreesContainer trees;
```
I think this will cut down on the number of variables and line count a bit